marine_sensor_msgs/msg/RadarSector.msg

```diff
@@ -0,0 +1,37 @@
+# A collection of marine radar rays, typically covering a contiguous
```

Comment:
In the video you posted on the discourse thread, it looked like you were updating the display more quickly than once a revolution. You also mentioned that the sensor will batch multiple rays into one UDP packet. So ... do you publish multiple RadarSectors within a complete 360-degree rotation? Or is this message always meant to contain a full rotation?

Reply:
Maybe copy some of the more verbose explanation into the message comments?